src/test/java/org/snomed/snowstorm/core/data/services/ConceptServiceTest.java

```diff
@@ -1699,6 +1703,138 @@ private boolean waitUntil(Supplier<Boolean> supplier, int maxSecondsToWait) {
 		}
 	}
 
+	/**
+	 * This test illustrates a 'module jump' scenario.
+	 */
+	@Test
+	void testJumpModule() throws ServiceException {
+		// Setup
+		String mainBranchPath = "MAIN";
+		String extensionBranchPath = "MAIN/SNOMEDCT-SE";
+		String fsnTerm = "Bacon and Eggs (food)";
+		String synonymTerm = "English Breakfast";
+		String swedishFsnTerm = "Bacon och ägg";
+		String swedishLanguageReferenceSet = "551231000005103";
+		String parentConceptId = "107658001";
+		String anotherConceptId = "116680003";
+		String anotherDescriptionId = "218680004";
+		String extensionModule = "45991000052106";
+		String refSetId="123459876000000000";
+
+		Map<String, String> intPreferred = Map.of(US_EN_LANG_REFSET, descriptionAcceptabilityNames.get(PREFERRED),
+				GB_EN_LANG_REFSET, descriptionAcceptabilityNames.get(PREFERRED));
+		Map<String, String> svPreferred = Map.of(swedishLanguageReferenceSet, descriptionAcceptabilityNames.get(PREFERRED));
+		Map<String, String> svAcceptable = Map.of(swedishLanguageReferenceSet, descriptionAcceptabilityNames.get(ACCEPTABLE));
+		String ci = "CASE_INSENSITIVE";
+		conceptService.batchCreate(Lists.newArrayList(new Concept(parentConceptId), new Concept(anotherConceptId)), mainBranchPath);
+
+		// Create international concept with two descriptions
+		Description fsnDesc = new Description(fsnTerm).setTypeId(FSN).setCaseSignificance(ci).setAcceptabilityMap(intPreferred)
+				.setLang("en").setLanguageCode("en").addLanguageRefsetMember(GB_EN_LANG_REFSET, Concepts.PREFERRED, true);
+		Description synonymDesc = 	new Description(synonymTerm).setTypeId(SYNONYM).setCaseSignificance(ci).setAcceptabilityMap(intPreferred)
+				.setLang("en").setLanguageCode("en").addLanguageRefsetMember(GB_EN_LANG_REFSET, Concepts.PREFERRED, true);
+
+		Concept concept = new Concept()
+				.addDescription(fsnDesc)
+				.addDescription(synonymDesc)
+				.addAxiom(new Relationship(ISA, parentConceptId));
+		concept = conceptService.create(concept, mainBranchPath);
+
+		// Add refset membership
+		Iterator<Description> descriptionIterator =	concept.getDescriptions().iterator();
```

Comment:
This is a very generic way to add a refset member to a description...I'm not sure what it would mean.   Is it possible to go via the route of setting the acceptability for some specific language refset and save that way?